web-api/src/documents/virusScanPdfLambda.js

```diff
@@ -9,13 +9,10 @@ const { genericHandler } = require('../genericHandler');
 exports.virusScanPdfLambda = event =>
   genericHandler(
     event,
-    async ({ applicationContext }) => {
-      const { documentId } = event.pathParameters || event.path || {};
-
-      return await applicationContext.getUseCases().virusScanPdfInteractor({
-        applicationContext,
-        documentId,
-      });
+    async () => {
```

Comment:
Is there a story logged to reactivate this later?

Reply:
Yes: https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/6053